Update Schwab to reflect support of SMS and better notes.

PBorocz opened the pull request:
Based on first-hand experience and verbal confirmation with Schwab today.

PBorocz committed:
Update Investing/Charles Schwab to add support of SMS and provide up-to-date notes.

Based on first-hand experience and verbal confirmation with Schwab today.

Carlgo11 commented on _data/investing.yml:
```diff
@@ -29,6 +29,7 @@ websites:
     url: https://www.schwab.com/
     img: charlesschwab.png
     tfa:
+      - sms
```
I don't see any mentions of SMS in the note?

PBorocz replied:
Thanks @Carlgo11. Agreed...I added the SMS flag at the last moment after noticing that it wasn't there (yet having used it personally for several years)...thus, didn't think about adding it to the note. I have done so now _and_ explicitly confirmed Symantec support by Fidelity.

PBorocz committed:
Add explicit comment regarding existing SMS-based capability.

I added SMS to the core Investing page for Schwab only as it seemed to be an oversight since SMS
has been supported by Schwab for many years...thus, I didn’t comment on it (and titled
the respective notes page as "Advanced TFA" to distinguish it from less robust TFA methods).

Based on the review comment, I’ve revised the notes to reflect SMS-based support and clarified
the use of Symantec for other sites (and verifying Fidelity’s use)
